### PR TITLE
Update tomcat_mgr_default_userpass.txt

### PR DESCRIPTION
**Pull request by cmaruti**
Updated tomcat manager credentials list

### CODE REVIEW

**Comment by bwatters-r7** [comment]
You may want to add these to the default username list and default password list?
https://github.com/rapid7/metasploit-framework/blob/master/data/wordlists/tomcat_mgr_default_pass.txt
https://github.com/rapid7/metasploit-framework/blob/master/data/wordlists/tomcat_mgr_default_users.txt
There's some duplication, but different scenarios require different sources.

**Comment by cmaruti** [comment]
@bwatters-r7 pull request updated

File: data/wordlists/tomcat_mgr_default_pass.txt [hunk 1]
```diff
@@ -5,3 +5,9 @@ root
 tomcat
 s3cret
 vagrant
+QLogic66
+password
+Password1
+changethis
+r00t
+toor
```
Comment (gwillcox-r7) [inline data/wordlists/tomcat_mgr_default_pass.txt:13]: Seem to be missing the following entries here from tomcat_mgr_default_userpass.txt:

```
password1

j2deployer
OvW*busr1
kdsxc
owaspba
ADMIN
xampp
QLogic66
```

File: data/wordlists/tomcat_mgr_default_users.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 admin
 manager
 role1
+role
 root
 tomcat
 both
+QCC
```
Comment (gwillcox-r7) [inline data/wordlists/tomcat_mgr_default_users.txt:8]: Missing following users from userpass.txt:

```
j2deployer
ovwebusr
cxsdk
ADMIN
xampp
```

**Review by gwillcox-r7 — CHANGES_REQUESTED** [review]

File: data/wordlists/tomcat_mgr_default_users.txt (hunk 2 above)
Comment (cmaruti) [reply data/wordlists/tomcat_mgr_default_users.txt:8]: Done

**Review by cmaruti — COMMENTED** [review]
done

File: data/wordlists/tomcat_mgr_default_pass.txt (hunk 1 above)
Comment (gwillcox-r7) [reply data/wordlists/tomcat_mgr_default_pass.txt:13]: Verified resolved in d1fb2a7bdef1e1246cd64d53f2daaa4259d4d9fb

File: data/wordlists/tomcat_mgr_default_users.txt (hunk 2 above)
Comment (gwillcox-r7) [reply data/wordlists/tomcat_mgr_default_users.txt:8]: Verified resolved in 0305983be2d354c76a586cf9db7f1dc10fccbd01

File: data/wordlists/tomcat_mgr_default_pass.txt [hunk 3]
```diff
@@ -5,3 +5,16 @@ root
...
+password1
+j2deployer
+OvW*busr1
+kdsxc
+owaspba
+ADMIN
+xampp
```
Comment (gwillcox-r7) [inline data/wordlists/tomcat_mgr_default_pass.txt:20]: Just missing the blank password here but I'll add this in myself

**Review by gwillcox-r7 — APPROVED** [review]

File: data/wordlists/tomcat_mgr_default_pass.txt (hunk 3 above)
Comment (gwillcox-r7) [reply data/wordlists/tomcat_mgr_default_pass.txt:20]: Oh nm it did exist, GitHub just weirdly didn't show it in the review :/

**Comment by gwillcox-r7** [comment]
# Release Notes
The wordlists for Tomcat Manager have been updated with new default usernames and passwords that can be used by various scanner and exploit modules when trying to find and exploit Tomcat Manager installations with default usernames and/or passwords.

**Pull request merged**